sdk/aqueduct/client.py

```diff
@@ -427,7 +427,7 @@ def integration(
                 metadata=integration_info,
             )
         elif integration_info.service == ServiceType.AWS:
-            dynamic_k8s_integration_name = "%s:k8s" % name
+            dynamic_k8s_integration_name = "%s:aqueduct_ondemand_k8s" % name
```

Comment:
Curious why this change?

Reply:
just to prevent this name from colliding with user-defined names. `aqueduct_ondemand_k8s` is much less likely to collide than `k8s`